WordPress/gutenberg
# Create Block: Add missing `viewScriptModule` field

Follow up #58211

## What?

I noticed that when specifying an interactivity template in the `create-block` package, the `viewScriptModule` was not added to `block.json`, so the block did not work as expected on the front end.

## Why?

In #58211, a field was added to the interactivity template side, but it appears that the field is not added to the create-block package that actually generates `block.json`.

## Testing Instructions

- Check out this PR.
- Run: `path/to/gutenberg/packages/create-block/index.js test-block --template path/to/gutenberg/packages/create-block-interactive-template`
- Check the `src/block.json` file.

### Before

```json
{
	"$schema": "https://schemas.wp.org/trunk/block.json",
	"apiVersion": 3,
	"name": "create-block/test-block",
	"version": "0.1.0",
	"title": "Test Block",
	"category": "widgets",
	"icon": "media-interactive",
	"description": "An interactive block with the Interactivity API",
	"example": {},
	"supports": {
		"interactivity": true
	},
	"textdomain": "test-block",
	"editorScript": "file:./index.js",
	"editorStyle": "file:./index.css",
	"style": "file:./style-index.css",
	"render": "file:./render.php"
}
```

### After

```json
{
	"$schema": "https://schemas.wp.org/trunk/block.json",
	"apiVersion": 3,
	"name": "create-block/test-block",
	"version": "0.1.0",
	"title": "Test Block",
	"category": "widgets",
	"icon": "media-interactive",
	"description": "An interactive block with the Interactivity API",
	"example": {},
	"supports": {
		"interactivity": true
	},
	"textdomain": "test-block",
	"editorScript": "file:./index.js",
	"editorStyle": "file:./index.css",
	"style": "file:./style-index.css",
	"render": "file:./render.php",
	"viewScriptModule": "file:./view.js"
}
```

Also run the following command and make sure that no unintended fields are added to block.json as before.

- `path/to/gutenberg/packages/create-block/index.js test-block`
- `path/to/gutenberg/packages/create-block/index.js test-block --template path/to/gutenberg/packages/create-block-tutorial-template`
